source/index.html.erb

```diff
@@ -3,6 +3,13 @@ pageable: true
 per_page: 10
 ---
 
+<script async src="https://www.googletagmanager.com/gtag/js?id=UA-76648418-8"></script>
+<script>
+window.dataLayer = window.dataLayer || [];
+function gtag(){dataLayer.push(arguments);} gtag('js', new Date());
+gtag('config', 'UA-76648418-8');
```

Comment:
In which account this code is registered?

Reply:
@naamaesa gave this code

Reply:
The code is provided to me by Camille from Analytics panel.
At any rate we will either see the stats or not.
Let's get this online.